testing/run-scripts/prometheus.yml

```diff
@@ -0,0 +1,21 @@
+# A scrape configuration scraping a Node Exporter, the Prometheus server
+# itself, and a container exposing zork metrics.
+scrape_configs:
+  # Scrape Prometheus itself every 5 seconds.
+  - job_name: 'prometheus'
+    scrape_interval: 5s
+    target_groups:
```

Comment:
Looks like this has been renamed `static_configs`:
https://github.com/prometheus/docs/blob/master/content/docs/operating/configuration.md#scrape_config